src/image-sync-manager.ts

```diff
@@ -64,16 +53,16 @@ export class ImageSyncManager {
     this._imagePlayer.preLoadImages();
   };
 
-  private _onTimedMetadata = () => {
+  private _onTimedMetadataChange = ({payload}: TimedMetadata) => {
     // TODO: use single "metadata" TextTrack once cue-point manager become use it
```

Comment:
let's remove the TODO cause active cue-points already comes from payload of `TIMED_METADATA_CHANGE` event